docs/io/ascii/read.rst

```diff
@@ -298,6 +303,15 @@ Guessing can be disabled in two ways::
   astropy.io.ascii.set_guess(False)                 # set default to False globally
   data = astropy.io.ascii.read(table)               # guessing disabled
 
+Debugging
+"""""""""
+
+In order to get more insight into the guessing process and possibly debug if
+something isn't working as expected, use the
+`~astropy.io.ascii.get_read_trace()` function.  This returns a traceback of the
+attempted read formats for the last call to `~astropy.io.ascii.read()` where
+guessing was enabled.
```

Comment:
It seems that this gets written even is guessing is *not* enabled?
https://github.com/astropy/astropy/pull/3688/files#diff-e6b2d1f1997a9b7e467819e2c98be409R251